frontend/src/views/web/invite-template.html

```diff
@@ -0,0 +1,21 @@
+<md-dialog>
+  <md-dialog-content class=ev-md-dialog-content>
+    <p class=ev-invite><strong>Invite others to this team</strong></p>
+    <p>Enter the email address of the person</p>
+    <form name=inputForm>
+      <md-input-container class=ev-md-input-container>
+        <input class=dark-autofill placeholder=email type=email ng-required=true size=25 name=email ng-model=model.value />
```

Comment:
you are having `ng-model="model.value"` and you're never using it in controllers. that's why invite is not woirking on clicking `Send Invite`.